app/src/main/java/de/westnordost/streetcomplete/quests/opening_hours/AddOpeningHours.kt

```diff
@@ -116,7 +117,7 @@ class AddOpeningHours(
           or barrier
           or amenity ~ toilets|bicycle_rental
         )
-        and opening_hours:signed != no
+        and (opening_hours:signed != no or (opening_hours:signed = no and opening_hours:signed older today -1 years))
```

Comment:
Why? doesn't this lead to this quest being asked also if not signed (instead of the CheckOpeningHoursSigned)?

Reply:
Can you add test cases that test:

- `isApplicableTo returns false if the opening hours check date is older than 1 year, while they are not signed` (i.e. no signed check date is set)
- `isApplicableTo returns false if last edit date is older than 1 year, while they are not signed` (not any check date is set)

Reply:
Tests added